pbcli.cfg

```diff
@@ -0,0 +1 @@
+'log="pbcli.log"
```

Comment:
Which other variables would you suggest to be placed in here? 
The only one that I thought about is base_url that is in http module.

Reply:
Any fixed variable thought enviroment will no change, to be sourced.
Ex.:
base_url -> Fixed base url for project
log_directory -> Fixed log directory
conn_timeout -> Connection timeout.